### PR TITLE
likelihood object always needs data, and sampler object does not need…

### PR DESCRIPTION
**Pull request by andreufont**
… free parameters

### CODE REVIEW

**Comment by andreufont** [comment]
@Chris-Pedersen - this is a simple PR to simplify a bit the code, by removing options that were useful during code development but that are not used anymore. The notebooks run fine (both compressed and primordial), but I could not test the script yet since I'm not sure about multiprocessing (see Slack).

File: lace/emulator/test_simulation.py [hunk 1]
```diff
@@ -14,7 +14,7 @@ class TestSimulation(object):
     in velocity units """
 
     def __init__(self,basedir,sim_label,skewers_label='Ns500_wM0.05',p1d_label="p1d",
-            z_max=4.0,kmax_Mpc=8,kp_Mpc=0.7,pivot_scalar=0.05):
```
Comment (Chris-Pedersen) [inline lace/emulator/test_simulation.py:17]: I think I'd prefer to keep this option here as this object is what I use to validate the emulator plots - its essentially a more concise way to read the p1d_archive values for a single simulation. As long as there is a sensible default value, we don't need to have one set when using it inside a data object.

Comment (andreufont) [reply lace/emulator/test_simulation.py:17]: @Chris-Pedersen - could you point me to the code used for this validation plots? Meanwhile I'll leave it as you requested, but I'd like to study alternatives long term.

Comment (Chris-Pedersen) [reply lace/emulator/test_simulation.py:17]: I haven't used these validation scripts for a while, so happy to leave this as is and we can revisit it at a later date!

**Comment by Chris-Pedersen** [comment]
Have tested this and find consistent results between this and master when running the sampler on Hypatia, so merging!

**Pull request merged**